app/models/in_person_enrollment.rb

```diff
@@ -136,8 +136,8 @@ def due_date
   end
 
   def days_to_due_date
-    today = DateTime.now
-    (today...due_date).count
+    today = Time.zone.now.utc
+    (((due_date - today) / 3600) / 24).to_i
```

Comment:
I think this can make use of ActiveSupport stuff for date math:
```suggestion
    (due_date - today).seconds.to_days
```

I'd agree with Andrew's suggestion as well

Reply:
So @mitchellhenke the result of the subtraction is a float object, which as far as I can tell does not have any of the ActiveSupport time methods (seconds, days) available on it.

Reply:
Ah sorry, the method is `in_days` rather than `to_days` (will edit original suggestion).

I'm not sure if that's what the method is after ultimately though. If `2023-10-30T23:59:59Z - 2023-10-31T00:00:00Z` is supposed to be `1` this method will be wrong. For strictly day-based math:

```ruby
(due_date.to_date - today.to_date).to_i
```

Reply:
@mitchellhenke I think your initial suggestion of `(due_date - today).seconds.in_days.to_i` is the correct one. I think it makes more sense to keep it in days.